streams/cl.m3u

```diff
@@ -385,11 +391,37 @@ http://159.69.56.148:25461/live/Mix24-7Gratis/78qy9/6.m3u8
 http://159.69.56.148:25461/live/Mix24-7Gratis/78qy9/7.m3u8
 #EXTINF:-1 tvg-id="" status="error",MIX 24/7 Puro Rock (720p)
 http://159.69.56.148:25461/live/Mix24-7Gratis/78qy9/25.m3u8
-#EXTINF:-1 tvg-id="" status="error",Camara de Diputados (DVR) (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="CDTV.cl" status="error",CDTV (DVR) (720p) [Not 24/7]
 http://camara.02.cl.cdnz.cl/cdndvr/ngrp:live_all/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="error",CHV Noticias (720p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCRsUoZYC1ULUspipMRnMhwg/live
 #EXTINF:-1 tvg-id="" status="error",Estación TV (Chillán) (720p) [Not 24/7]
 https://pantera1-100gb-cl-movistar.dps.live/estaciontv/estaciontv.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="error",Spectrum Channel LGBTQ+ (480p) [Not 24/7]
+#EXTINF:-1 tvg-id="SpectrumChannel.cl" status="error",Spectrum Channel (480p) [Not 24/7]
```

Comment:
Now named SMAS, under the spectrumchannel.cl website.

Reply:
This seems similar with Energeek and Kanade. They come together but then split up and go their separate ways.